### PR TITLE
FALCON-1888 : Falcon JMS notification details and example

### PR DESCRIPTION
**Pull request by vramachan**
Updates the operability document to enable Falcon user notifications and example code to read and Map Message from ActiveMQ.

### CODE REVIEW

File: docs/src/site/twiki/Operability.twiki [hunk 1]
```diff
@@ -73,21 +73,113 @@ or send alerts according to their requirements.
...
+The name of the JMS topic is same as the process/feed name. Falcon sends a map message for every feed instance that is
+created/deleted/replicated/imported/exported to the JMS topic. The JMS Map Message sent to a topic has the following
+fields:
+
+   1. cluster - name of the current cluster the feed/process is dependent on.
+   1. entityType - type of the entity (feed or process).
+   1. entityName - name of the entity.
+   1. nominalTime - instance time (or data date).
+   1. operation - operation like generate, delete, replicate, import, export.
+   1. feedNames - name of the feeds which are generated/replicated/deleted/imported/exported.
```
Comment (sowmyaramesh) [inline docs/src/site/twiki/Operability.twiki]: Minor nit: Did you intend to bullet them? I can see 1. for all the points.

File: docs/src/site/twiki/Operability.twiki [hunk 2]
```diff
@@ -73,21 +73,113 @@ or send alerts according to their requirements.
...
-entityName, feedNames, feedInstancePath, workflowId, runId, nominalTime, timeStamp, brokerUrl, brokerImplClass, entityType, operation, logFile, topicName, status, brokerTTL;
-
-For a given feed that is scheduled, the name of the topic is same as the feed name.
-Falcon sends a map message for every feed instance that is deleted/archived/replicated depending upon the retention policy set in the feed definition.
-The JMS !MapMessage sent to a topic has the following properties:
-entityName, feedNames, feedInstancePath, workflowId, runId, nominalTime, timeStamp, brokerUrl, brokerImplClass, entityType, operation, logFile, topicName, status, brokerTTL;
-
-The JMS messages are automatically purged after a certain period (default 3 days) by the Falcon JMS house-keeping service.TTL (Time-to-live) for JMS message
-can be configured in the Falcon's startup.properties file.
+The implementation class and the broker url of the JMS engine are specified in the cluster definition associated
```
Comment (sowmyaramesh) [inline docs/src/site/twiki/Operability.twiki]: By default, Falcon embeds ActiveMQ but user can also run external standalone ActiveMQ. Can we mention this in the documentation?

File: docs/src/site/twiki/Operability.twiki (hunk 1 above)
Comment (vramachan) [inline docs/src/site/twiki/Operability.twiki]: this is twiki format - after build, the numbers will show up on the html

File: docs/src/site/twiki/Operability.twiki (hunk 2 above)
Comment (vramachan) [inline docs/src/site/twiki/Operability.twiki]: details added on the system notification - though internal.

**Comment by peeyushb** [comment]
Patch looks good. +1.

**Pull request closed without merging**